### PR TITLE
build: Update jsonnet to 0.20.0

### PR DESCRIPTION
**Pull request by jachym-tousek-keboola**

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (jachym-tousek-keboola) [inline .github/workflows/ci.yml:14]: Pipeline was failing on Go 1.17 but I don't think we care about such old version. If anything we might want to add newer go versions into the ci matrix.

File: .github/workflows/ci.yml
Comment (jachym-tousek-keboola) [inline .github/workflows/ci.yml:94]: Doesn't work with latest goreleaser. I don't think it's our responsibility to debug the issues and make it work so I just downgraded goreleaser to the version that was latest at the time when jsonnet 0.20 released. We can remove the version constraint when jsonnet releases another version where the maintainers will need to fix this.

File: Makefile
Comment (jachym-tousek-keboola) [inline Makefile:17]: I have no clue what's the deal with bazel but I found that you disabled the bazel step in CI here with a note that it doesn't work in the fork:
https://github.com/keboola/go-jsonnet/blob/24a6732c0397cb6cdf9a1ab77df3fa9240c51b9d/.github/workflows/ci.yml#L50

It was still being executed in `make all` though and ofc failed. Can you share some details what's wrong with bazel in forked repo? Should we try to fix it or do we not care? It might simply be same as goreleaser that current latest bazel no longer works but older might. Figured I should ask you for a comment on this first though since I have no experience with bazel.

Comment (michaljurecko) [reply Makefile:17]: `bazel` builds a `C library` from the Go code, it is out of the our scope

**Review by michaljurecko — APPROVED** [review]
LGTM

Before the merge, please try, that it works with the KAC repo (no test fails).

**Comment by jachym-tousek-keboola** [comment]
Yeah it's passing so I'm merging this. :)

**Pull request merged**
